site/hxapi/hotel/index.md

```diff
@@ -14,6 +14,7 @@ Here are the hotel specific endpoints:
  | ------                                     | --------                                                                         | ------ |
  | Availability at hotel      | [https://api.holidayextras.co.uk/v1/hotel/LocationCode](av)            | GET    |
  | Make booking at hotel      | [https://api.holidayextras.co.uk/v1/hotel/LocationCode](bkg)           | POST   |
+ | Pre-booking price check  | [https://api.holidayextras.co.uk/v1/hotel/LocationCode/priceCheck](priceCheck) | GET    |
```

Comment:
so we would get the recent prices for all products at a location and also get the recent prices for a single location. 

I guess it makes sens the order of the params in the path as the price check is for a location/product. 
What would you say about leaving the locationCode/productCode at the end of the path and having the url like

<baseUrl>/v1/hotel/priceCheck/LocationCode 

The benefit would be in the controller when we define which controller goes is used on which route. As in future we might have /v1/hotel/something-else-that-is-not-a-product-code/action and defining this will need a regular expression that is well tested versus if we'd have v1/hotel/priceCheck/LocationCode

Reply:
Re: your first sentence, that's a good point. It's not made so explicit with hotels as it is with parking.
Although you should be able to do availability https://api.holidayextras.co.uk/v1/hotel/LHR for an airport and also https://api.holidayextras.co.uk/v1/hotel/LHRHIL for a hotel, I'm not sure that makes sense for a price check. Since it won't be cached, we don't want people overusing it.
So it should only be called with actual hotel product codes, not with airport codes (just as should be the case for the booking endpoint already).